.pre-commit-config.yaml

```diff
@@ -1,6 +1,4 @@
 default_stages: [commit]
-default_language_version:
-  python: python3.8
```

Comment:
Would we have locally working pre-commits in the future considering that 3.8 is version for currently standard Ubuntu 20?

My expectation that without this setting chore will push individual pre-commits to version that will no longer will be compatible with python3.8

Reply:
This won't be an issue. This setting is only about the pre-commit version that is used for the validation of this repo. We still have a Python 3.8 CI job that makes sure the code we ship works on 3.8.